overlays/python-imports-check-typo.nix

```diff
@@ -0,0 +1,33 @@
+{ builtAttrs
+, packageSet
+, namePositions
+}@attrs:
+
+final: prev:
+let
+  inherit (prev) lib;
+  inherit (import ../lib { inherit lib; }) checkFor; 
+
+  incorrectSpellings = [
+    "pythonImportTests"
```

Comment:
This will never be a complete list and is a deeper issue in nix. I don't want to start something with levenshtein because a good reviewer should catch that.

Reply:
Adding a levehnstein checker is on my [to do list](https://github.com/jtojnar/nixpkgs-hammering/issues/1) since typos are common and tooling should catch them. Example from today https://discourse.nixos.org/t/error-building-python-package-no-matching-distribution-found-could-not-find-a-version-that-satisfies-the-requirement-six/11143